testing-common/src/test/groovy/muzzle/ReferenceCollectorTest.groovy

```diff
@@ -102,65 +110,33 @@ class ReferenceCollectorTest extends Specification {
     setup:
     def collector = new ReferenceCollector({ false })
     collector.collectReferencesFromAdvice(TestClasses.InvokeDynamicAdvice.name)
+    collector.prune()
     def references = collector.getReferences()
 
     expect:
     references['muzzle.TestClasses$MethodBodyAdvice$SomeImplementation'] != null
     references['muzzle.TestClasses$MethodBodyAdvice$B'] != null
   }
 
-  def "should create references for helper classes"() {
```

Comment:
Hmm, instead of removing this test WDYT about moving `HelperInterface` out to a "library" package? This test verifies that helper classes are properly collected for references and that private/static/constructor methods are skipped.

Reply:
oh, I didn't have to remove this test 🤦‍♂️